tests/cli_ctl_environment.yml

```diff
@@ -0,0 +1,64 @@
+name: Environement Command TestSuite with CDS ctl
+testcases:
+- name: assert filepath, your current directory must be at the root of this project
+  steps:
+  - script: '[ -f ./tests/fixtures/group_to_add.yml ]'
+  assertions:
+    - result.code ShouldEqual 0
+- name: prepare test
+  steps:
+  - script: {{.cds.build.cds}} project remove --force ITCLIAPPPRJ
```

Comment:
use cdsctl ?

Reply:
next pr see this branch https://github.com/ovh/cds/tree/feat_cli_ctl